core/storage/feedback/gae_models_test.py

```diff
@@ -0,0 +1,124 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and 
+# limitations under the License.
+
+__author__ = 'Shantanu Bhowmik'
+
+from core.platform import models
+(base_models, feedback_models)=models.Registry.import_models(
+    [models.NAMES.base_model, models.NAMES.feedback])
+import test_utils
+
+
+CREATED_ON_FIELD = 'created_on'
+
+
+class SuggestionModelTest(test_utils.GenericTestBase):
+    """Tests the SuggestionModel class."""
+
+    def setUp(self):
+        super(SuggestionModelTest, self).setUp()
+        base_models.BaseModel.created_on._auto_now = False
+        base_models.BaseModel.last_updated._auto_now = False
+        feedback_models.SuggestionModel.create('exp_id1', 'thread_id1',
+                                               'author_id', 1, 'state_name',
+                                               {'old_content': {}})
+        feedback_models.SuggestionModel.create('exp_id1', 'thread_id2',
+                                               'author_id', 1, 'state_name',
+                                               {'old_content': {}},
+                                               feedback_models.STATUS_ACCEPTED)
+        feedback_models.SuggestionModel.create('exp_id2', 'thread_id2',
+                                               'author_id', 1, 'state_name',
+                                               {'old_content': {}})
+
+    def get_suggestion_models_for_test(self, suggestions_list):
+        updated_suggestions_list = []
+        for suggestion in suggestions_list:
+            suggestion_dict = suggestion.to_dict()
+            if CREATED_ON_FIELD in suggestion_dict:
+                suggestion_dict.pop(CREATED_ON_FIELD)
+            updated_suggestions_list.append(suggestion_dict)
+        return updated_suggestions_list
+
+    def test_create_success(self):
```

Comment:
The intention is not to test get_by_exp_id. It is to test if the object was actually created. Note that the setUp() method calls the create method, which is what is being tested. Let me know if I need to change the approach here, but I do believe this is good to have.